crates/sui-cost/tests/snapshots/empirical_transaction_cost__good_snapshot.snap

```diff
@@ -24,17 +24,17 @@ expression: common_costs
     "storage_rebate": 0
   },
   "SharedCounterIncrement": {
-    "computation_cost": 197,
+    "computation_cost": 198,
```

Comment:
@oxade, this PR adds a field to a `MoveObject`'s `Owner` field (the version it was shared at).  Is it expected for the computation cost to go up (and for the storage-related costs *not* to change)?